block/sync.go

```diff
@@ -71,18 +68,16 @@ func (m *Manager) SettlementSyncLoop(ctx context.Context) error {
 
 				settlementBatch, err := m.SLClient.GetBatchAtHeight(m.State.NextHeight())
 				if err != nil {
-					return fmt.Errorf("retrieve batch: %w", err)
+					m.logger.Error("retrieve SL batch", "err", err)
+					break
 				}
 				m.logger.Info("Retrieved state update from SL.", "state_index", settlementBatch.StateIndex)
 
 				err = m.ApplyBatchFromSL(settlementBatch.Batch)
 				if err != nil {
+					m.freezeNode(context.Background(), err)
```

Comment:
why do we freeze node vs returning error and handling on the call site?

Reply:
updated it